pkg/model/components/apiserver.go

```diff
@@ -133,8 +133,21 @@ func (b *KubeAPIServerOptionsBuilder) BuildOptions(o interface{}) error {
 
 	c.AllowPrivileged = fi.Bool(true)
 	c.ServiceClusterIPRange = clusterSpec.ServiceClusterIPRange
-	c.EtcdServers = []string{"http://127.0.0.1:4001"}
-	c.EtcdServersOverrides = []string{"/events#http://127.0.0.1:4002"}
+	c.EtcdServers = nil
+	c.EtcdServersOverrides = nil
```

Comment:
Are these actually needed? Should be nil by default.

Reply:
I understand now :). Builder runs until there are no changes between runs, but without clearing the object first.

Reply:
Yeah exactly...showed up in tests :)

Reply:
I'll pretend it never happened :D